### ISSUE TITLE
MC: Refuse Position Control

### ISSUE THREAD

**Issue by tops4u** [PX4/Firmware#7989]
**Bug Report**

Powered Up Vehicle and placed it outdoors for some minutes. The Vehicle is equipped with a uBlox M8N GPS from Drotek which usually has a GPS lock within seconds. Then I changed EKF_RNG_AID to 1 and rebooted the vehicle. Then I placed it on the starting point and took of in Stabilized Mode. After switching to Pos Ctl I observed that the Vehicle drifted away but hold Altitude. I figured that PosCtl was refused without giving further details as to why. After watching the Logfile I observed, that there is no GPS Information available. 

1. Log: https://review.px4.io/plot_app?log=f4a5e794-6ff2-40a8-8ac3-2b89aa7e501a

I then switched this Parameter Back to 0 as this was the only change since the last successful flight, and it behaved again as expected:

2. Log: https://review.px4.io/plot_app?log=11e8986d-b0f4-4566-8a58-3706a1464ab4

How come? Even after a FC Reboot the GPS should keep the lock. To me it looks as the GPS was completely ignored in the 1. Log, not even amount of Sats is available. Despite this the Log has correct Timestamp, so GPS Information must have been available at some point.

**Comment by tops4u** [comment]
Today I had the same issue on a flight. This time I checked that there GPS indicated a fix (green blinking), but PX4 did not recognize it and also did not take the timestamp for logging for ex. 

Log File: https://logs.px4.io/plot_app?log=49ec42e2-ab8d-4217-a825-9cbaf4dc0d2a 

Connector is a standard Pin-Header Connector. GPS Status in Nuttx was fail / Protocol ubx. 

I have now upgraded to current Master. Once GPS is found it seems ok even after several reboots but was not recognized even after rebooting. 

Funny though is that the CBRK for GPS is not disabled. Should this not prevent Arming?
